src/utils/mailClient.js

```diff
@@ -0,0 +1,12 @@
+import { NotifyClient } from 'notifications-node-client'
+
+import dotenv from 'dotenv'
+dotenv.config()
+
+const notifyClient = new NotifyClient(process.env.GOVUK_NOTIFY_API_KEY || 'test-key')
+
+export const sendEmail = async (emailAddress, templateId, personalisation) => {
```

Comment:
A few issues with this function:

- it does not provide any functionality beyond `notifyClient.sendEmail(...).then(console.log, console.error)` which is not a bad thing itself except it makes it less useful in a way (see next point) 
- the caller has no way to tell whether the call succeeded or not
- none of the above facts are documented (I like the tooltips to tell me if I need to worry about catching errors etc)
- it needlessly changes the signature (leaving the third argument as an object would allow to easily pass more things in the future)
- the first two point make most of the test code redundant (e.g. since this is just a thin wrapper around the `NotifyClient.sendEmail()` method, there's not much to test, and that method is already tested in `notifications-node-client`)
- the second point makes testing awkward (by having to mock the console)

Not swallowing exceptions and return values, passing an explicit `resolved` handler
like `sendEmail(templateId, emailAddress, {personalisation, resolved: (result) => { console.info(result); return result;})` and explicitly catching failures via `sendEmail(...).catch(console.error)` in the controller would take care most of the issues and allow you to nuke almost all the new test code 🙂 (which consists mostly of setting up mocks etc)